tasmota/my_user_config.h

```diff
@@ -881,6 +881,7 @@
 //  #define VINDRIKTNING_SHOW_PM1                  // Display undocumented/supposed PM1.0 values
 //  #define VINDRIKTNING_SHOW_PM10                 // Display undocumented/supposed PM10 values
 //#define USE_LD2410                               // Add support for HLK-LD2410 24GHz smart wave motion sensor (+3k7 code)
+//#define USE_LD2410S                              // Add support for HLK-LD2410S 24GHz smart wave motion sensor (+4k6 code)
```

Comment:
Hello @arendst 
I can't find this string in "development" branch. It's correct?

Reply:
Yes. Correct. Update your branch.

Reply:
![image](https://github.com/user-attachments/assets/01d7d8a1-5693-4908-8bbe-b825967e732e)
The default branch (development) in project https://github.com/arendst/Tasmota